packages/dso-toolkit/src/components/annotation/annotation.models.ts

```diff
@@ -1,7 +1,117 @@
-import { AnnotationButton } from "../annotation-button/annotation-button.models";
-import { AnnotationOutput } from "../annotation-output/annotation-output.models";
+export type Annotation =
+  | AnnotationActiviteit
+  | AnnotationGebiedsaanwijzing
+  | AnnotationOmgevingsnorm
+  | AnnotationWerkingsgebied;
 
-export interface Annotation<TemplateFnReturnType> {
-  annotationButton: AnnotationButton;
-  annotationOutput: AnnotationOutput<TemplateFnReturnType>;
+/**
+ * Basis model voor de verschillende annotaties.
+ */
+interface AnnotationBase {
+  /**
+   * Een URL naar de verbeelding van de legenda.
```

Comment:
Is dit commentaar correct?

Reply:
Nee 🤪 

De property overal weggehaald en aangepast naar slot documentatie:

```
/**
 * @slot symbool - Een optionele afbeelding die de annotatie symboliseert.
 */
@Component({
  [...]
```